project/generate-dockerfile.sh

```diff
@@ -0,0 +1,266 @@
+#!/bin/bash
+set -e
+
```

Comment:
Can we please avoid commenting on the PR contents until we've made a decision on whether this lives on as-is or whether we shift gears again?  I'd like to avoid wasting @brahmaroutu's time any further than it has been, if we can. :wink: